transaction/interfaces.py

```diff
@@ -22,12 +22,19 @@ class ITransactionManager(Interface):
     """
 
     def begin():
-        """Begin a new transaction.
+        """Explicitly begin a new transaction.
 
         If an existing transaction is in progress, it will be aborted.
 
-        The newTransaction() method of registered synchronizers is called,
+        The ``newTransaction`` method of registered synchronizers is called,
         passing the new transaction object.
+
+        Note that transactions may be started implicitly without
+        calling ``begin``. In that case, ``newTransaction`` isn't
```

Comment:
We **do** _think_ of them as one and the same, but the current behavior betrays that model.